windows-gaming-driver/src/driver/mod.rs

```diff
@@ -99,18 +99,17 @@ pub fn run(cfg: &Config, tmp: &Path, data: &Path) {
 
     core.run(qemu.join(joined.or_else(|x| { error!("Unexpected error: {}", x); Ok(()) }))).expect("Unexpected error");
 
-	info!("unbinding resetable vfio-things");
+	info!("unbinding resettable vfio-things");
 	
-    for slot in cfg.machine.vfio_slots.iter() {
-		if let &VfioDevice::Temporarily(ref device) = slot {
-			let mut child = Command::new(Path::new(::DATA_FOLDER).join("vfio-ubind")).arg(device).arg("-r").spawn().expect("failed to run vfio-ubind");
+    for dev in cfg.machine.vfio_slots.iter() {
+		if dev.resettable {
+			let mut child = Command::new(data.join("vfio-ubind")).arg(&dev.slot).arg("-r").spawn().expect("failed to run vfio-ubind");
 			match child.wait() {
 				Ok(status) => 
 						if !status.success() {	
 						error!("vfio-ubind failed with {}! The device might still be bound to the vfio-driver!", status);
 						},
```

Comment:
Use match guards here to include the if condition into the match arm. This does add a new match arm, but makes the code more readable.